spec/models/wishlist_item_spec.rb

```diff
@@ -14,9 +14,12 @@
 
 describe WishlistItem do
   context 'relationships' do
-    it { is_expected.to belong_to(:wishlist) }
-    it { is_expected.to belong_to(:item) }
-    it { is_expected.to have_many(:pledges).dependent(:destroy) }
+    pledge_association = WishlistItem.reflect_on_association(:pledges)
+
+    it { WishlistItem.reflect_on_association(:wishlist).macro.should eq(:belongs_to) }
```

Comment:
Tools like `#macro` and `#reflect_on_association` are powerful, but they are confusing to people not intimately familiar with rspec and can lead to strange errors. For easier maintenance/onboarding, we're trying to keep to the basic rspec API.

Also, when not specifically using [one-liner syntax](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax) (i.e. `it { is_expected.to ... }` or `it { should ... }`), feel free break this into multiple lines with the `do ... end` syntax. That has a few advantages: (a) more descriptive to read, (b) allows you to add a title, making spec output much more helpful.